lib/openstax/aws/secrets_specification.rb

```diff
@@ -13,11 +13,12 @@ def self.from_file_name(file_name:, format:, top_key: nil, preparser: nil)
       new(content: content, format: format, top_key: top_key, preparser: preparser)
     end
 
-    def self.from_git(org_slash_repo:, sha:, path:, format:, top_key: nil, preparser: nil)
+    def self.from_git(org_slash_repo:, sha:, path:, github_token:, format:, top_key: nil, preparser: nil)
```

Comment:
Same here, better make the token optional in case some repo already calls this method directly.